pkg/operator/ceph/object/rgw.go

```diff
@@ -341,18 +341,25 @@ func GetStableDomainName(s *cephv1.CephObjectStore) string {
 }
 
 func getDomainName(s *cephv1.CephObjectStore, returnRandomDomainIfMultiple bool) string {
+	endpoints := []string{}
 	if s.Spec.IsExternal() {
 		// if the store is external, pick a random endpoint to use. if the endpoint is down, this
 		// reconcile may fail, but a future reconcile will eventually pick a different endpoint to try
-		endpoints := s.Spec.Gateway.ExternalRgwEndpoints
-		idx := 0
-		if returnRandomDomainIfMultiple {
-			idx = rand.Intn(len(endpoints)) //nolint:gosec // G404: cryptographically weak RNG is fine here
+		for _, endpoint := range s.Spec.Gateway.ExternalRgwEndpoints {
+			endpoints = append(endpoints, endpoint.String())
 		}
```

Comment:
Where is the code to extract the domain name from the endpoint here? This function is supposed to return a domain name.